content/shared/v3-core-plugins/_index.md

```diff
@@ -654,6 +537,7 @@ docker exec -it CONTAINER_NAME influxdb3 install package pandas
 This creates a Python virtual environment in your plugins directory with the specified packages installed.
 
 {{% show-in "enterprise" %}}
+
 ### Connect Grafana to your InfluxDB instance
 
 When configuring Grafana to connect to an InfluxDB 3 Enterprise instance:
```

Comment:
You don't need to do this now, but we should:
- Generalize this recommendation beyond Grafana
- Add it to a list of multi-node recommendations (not specific to the processing engine)